app/javascript/src/components/Property/PropertyDetails.css

```diff
@@ -0,0 +1,20 @@
+.hackney-property-alerts {
+	width: 60%;
+	padding: 0;
+}
+  
+.hackney-property-alerts li {
+	  font-family: Arial, Helvetica, sans-serif !important;
+	  text-transform: capitalize;
+	  margin-bottom: 5px;
+	  padding: 4px 8px;
+	  font-weight: normal;
+}
+
+.bg-orange {
+	  background-color: #FFA300;
+}
+
+.bold-text {
+	font-weight: bold;
+}
```

Comment:
This looks good. You can improve it by:

- Moving the styling into `app/javascript/stylesheets/components`?
- You can also change the file to `.scss` which might make the styling cleaner, as well as use 2 space on your tabs

Reply:
Just to add there are also gov-uk css classes that we should try to use instead of making our own custom css if possible https://design-system.service.gov.uk/styles/typography/#font-weight